examples/scenes/src/mmark.rs

```diff
@@ -0,0 +1,191 @@
+//! A benchmark based on MotionMark 1.2's path benchmark.
```

Comment:
Should we include a sharing link (e.g. https://browserbench.org/MotionMark1.2/developer.html?warmup-length=2000&warmup-frame-count=30&first-frame-minimum-length=0&test-interval=15&display=minimal&tiles=big&controller=adaptive&frame-rate=50&time-measurement=performance&suite-name=MotionMark&test-name=Paths&complexity=1)

Caveating it as discussed with numbers only approximately comparable